### PR TITLE
#234 IT tests for Table view

### PR DESCRIPTION
**Pull request by Vatavuk**
For #234, created IT tests for create/edit/delete. Found one bug in `ModelForm` which prevents Table form to be saved so I've put `skip` annotations on tests and left a puzzle
with detail explanation what has to be fixed.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#322](https://github.com/timelesslounge/timelessis/issues/322) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#322](https://github.com/timelesslounge/timelessis/issues/322) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/b934b569-9b74-40a9-a703-058e220197df); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/restaurants/tables/views.py [hunk 1]
```diff
@@ -44,8 +44,9 @@ def create():
     """ Create new table """
     form = forms.TableForm(request.form)
     if request.method == "POST" and form.validate():
+        print("FORM", form.data)
```
Comment (vladarefiev) [inline timeless/restaurants/tables/views.py]: @Vatavuk remove it pls

File: timeless/restaurants/models.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ class TableReservation(DB.Model):
     reservation = DB.relationship("Reservation", back_populates="tables")
 
 
-class Table(PosterSyncMixin, DB.Model):
+class Table(TimestampsMixin, PosterSyncMixin, DB.Model):
```
Comment (vladarefiev) [inline timeless/restaurants/models.py]: @Vatavuk I look at this and realize it's useless to add this mixin to every model with other mixins. I think this mixin will be used in 90% models, may be create an abstract Model like this: https://docs.sqlalchemy.org/en/latest/orm/extensions/declarative/mixins.html#augmenting-the-base
WSYT? If you are agree, let's make a puzzle

File: tests/integration/it_tables_test.py [hunk 3]
```diff
@@ -1,30 +1,89 @@
...
+        width=100,
+        height=75,
+        status=1,
+        max_capacity=4,
+        multiple=False,
+        playstation=True
+    )
+    db_session.add(table)
+    db_session.commit()
+    response =  client.post(url_for("table.delete", id=1))
```
Comment (vladarefiev) [inline tests/integration/it_tables_test.py]: @Vatavuk I see two spaces

File: tests/integration/it_tables_test.py [hunk 4]
```diff
@@ -1,30 +1,89 @@
...
+ from tests.
 """
 import pytest
+
 from http import HTTPStatus
 
+from flask import url_for
+
+from timeless.restaurants.models import Table
+
```
Comment (vladarefiev) [inline tests/integration/it_tables_test.py]: `http` is python native lib.
`pytest, flask` are thirdparty
now it should be in this way:
```
from http import HTTPStatus

import pytest
from flask import url_for

from timeless.restaurants.models import Table
```

**Comment by vladarefiev** [comment]
@Vatavuk really good to me. but a few small comments

File: timeless/restaurants/models.py (hunk 2 above)
Comment (Vatavuk) [reply timeless/restaurants/models.py]: @vladarefiev couldn't agree more

**Comment by Vatavuk** [comment]
@vladarefiev fixed

**Comment by vladarefiev** [comment]
@Vatavuk pls fix conflicts and go to merge

**Comment by Vatavuk** [comment]
@vladarefiev merged with master

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Job [#322](https://github.com/timelesslounge/timelessis/issues/322) is not in the agenda of @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), can't inspect

**Comment by 0crat** [comment]
The job [#322](https://github.com/timelesslounge/timelessis/issues/322) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>Messages in a job [always](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html#4-avoid-noiseaddress-your-comments) start with a name of a user they are addressed to.

Please correct your [message](https://github.com/timelesslounge/timelessis/pull/322#discussion_r258440365) by indicating an addressee in the beginning.

**Comment by vladarefiev** [comment]
@ypshenychka thanks, fixed

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**